dir.targets

```diff
@@ -81,11 +87,11 @@
          with state.  They were passing because MSTest was serial. -->
 
     <!-- Run tests on the full framework -->
-    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.x86.exe @(MainAssembly, ' ') -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html $(XunitOptions) > %(MainAssembly.FullPath)_stdout.txt"
+    <Exec Command="$(PackagesDir)\xunit.runner.console.$(XunitVersion)\tools\xunit.console.exe @(MainAssembly, ' ') $(XunitOptions) -noshadow -parallel none -xml %(MainAssembly.FullPath)_TestResults.xml -html %(MainAssembly.FullPath)_TestResults.html > %(MainAssembly.FullPath)_stdout.txt"
```

Comment:
Please tell me why. It seems to work fine. I was not sure why we used x86 version so I changed it.